app.py

```diff
@@ -2,23 +2,36 @@
 from pathlib import Path
 from random import randint
 from tempfile import NamedTemporaryFile
-from typing import List, Any
+from typing import Any, List
 
 import ifcopenshell
-import ifcopenshell.geom as geom
```

Comment:
ifcopenshell package does not import submodules explicitly, so you should import this subpackage: `import ifcopenshell.geom`